src/main/java/org/jboss/ejb/client/EJBClient.java

```diff
@@ -64,6 +64,11 @@ private EJBClient() {
     public static final String SOURCE_ADDRESS_KEY = "jboss.source-address";
 
     /**
+     * A JNDI context key which, if defined, identifies that the proxy to be created should have affinity to the cluster.
+     */
+    public static final String CLUSTER_AFFINITY = "cluster-affinity";
```

Comment:
I can merge this as an interim commit, but this property should be qualified with "jboss.", and...